src/oss/confluent.sh

```diff
@@ -749,8 +749,12 @@ config_control-center() {
 }
 
 export_control-center() {
-    #no-op
-    return
+    get_service_port "listeners" "${confluent_conf}/control-center/control-center.properties"
```

Comment:
should we match something more specific and `listeners`?

Reply:
this is what everyone else does. soo.... 🤷‍♂️